pkg/flag/rktflag_test.go

```diff
@@ -124,112 +125,97 @@ func TestBitFlags(t *testing.T) {
 		if err := bf.Set(tt.opts); (err != nil) != tt.parseErr {
 			t.Errorf("test %d: Could not parse options as expected: %v", i, err)
 		}
-		if tt.ex >= 0 && bf.hasFlag(tt.ex) == tt.logicErr {
+		if tt.ex >= 0 && bf.HasFlag(tt.ex) == tt.logicErr {
 			t.Errorf("test %d: Result was unexpected: %d != %d",
-				i, tt.ex, bf.flags)
+				i, tt.ex, bf.Flags)
 		}
 	}
 }
 
-func TestSecFlags(t *testing.T) {
+type ss []string
+type sm map[string]string
```

Comment:
these are just shorthand types used for testing code- stringmap and stringslice.